### PR TITLE
Adds testing for Description

### PR DESCRIPTION
**Pull request by HelenaLykstoft**
- Unit test for description part

### CODE REVIEW

File: Validator/Validator.cs [hunk 1]
```diff
@@ -4,8 +4,16 @@ public class Validator
...
-        return true;
+        if (description.Length < 5 || description.Length > 255)
+        {
+            return false;
+        }
+        else if (description.Length >= 5 && description.Length <= 255)
+        {
+            return true;
+        }
+        return false;
```
Comment (thejamiegc) [inline Validator/Validator.cs]: jeg mener at der ikke er behov for else if. Da if statement dækker alle false scenarier. Dermed er alle andre scenarier true.

File: Tests/Validator_test/ValidatorTest.cs [hunk 2]
```diff
@@ -12,19 +12,51 @@ public ValidatorTest()
...
 
         // Act
         bool result = _validator.ValidateDescription(validDescription);
 
         // Assert
         Assert.True(result);
     }
+    
+    [Fact]
+    public void ValidateDescription_ShouldReturnFalse_WhenDescriptionIsInvalid()
```
Comment (thejamiegc) [inline Tests/Validator_test/ValidatorTest.cs]: Denne test er fin men jeg ville være opmærksom på boundary values.

File: Tests/Validator_test/ValidatorTest.cs [hunk 3]
```diff
@@ -12,19 +12,51 @@ public ValidatorTest()
...
+    public static IEnumerable<object[]> GetDescriptionTestData()
+    {
+        yield return new object[] { "Test", false }; // 4 characters
+        yield return new object[] { "Tests", true }; // 5 characters
+        yield return new object[] { new string('a', 255), true }; // 255 characters
+        yield return new object[] { new string('a', 256), false }; // 256 characters
+    }
+
+    [Theory]
+    [MemberData(nameof(GetDescriptionTestData))]
```
Comment (thejamiegc) [inline Tests/Validator_test/ValidatorTest.cs]: Som jeg forstår dette går det i mod best practice af test da der er noget logik i objekterne som gøre at du kan både teste for valid og invalid scenarier inputs samtidig. så jeg ville lave det mere simplet.

**Review by thejamiegc — CHANGES_REQUESTED** [review]
jeg nogen forslag til hvordan jeg ville gøre koden bedre og så det mere forstålig.

**Review by thejamiegc — APPROVED** [review]
perfekt!

**Review by solskinIsak — APPROVED** [review]
